deploy/examples/create-external-cluster-resources.py

```diff
@@ -693,6 +699,13 @@ def get_ceph_external_mon_data(self):
                 "Only 'v2' address type is enabled, user should also enable 'v1' type as well"
             )
         ip_port = str(q_leader_details["public_addr"].split("/")[0])
+
+        if self._arg_parser.v2_port_enable:
+            if q_leader_addrvec[0]["type"] == "v2":
+                ip_port = q_leader_addrvec[0]["addr"]
```

Comment:
` ip_port = q_leader_addrvec[0]["addr"]`` 
 ```
 v2port=3300
 Ip_port= v2_port
 ```

Reply:
The thing is, `ip_port` doesn't only have the port number it has the ip addr as well, that's the reason I am saving the value directly from the `addr` property.

Reply:
So can you renaming that to smtg like full ipadress

And also can you think if there would be any issue with IPV6 address

Reply:
Updated the variable name to `ip_addr`
I don't think there would be any issue with IPv6 addresses, the `split` function could have caused issues in the `tests` but I have updated that as well.